### DISCUSSION TITLE
Semestral

### DISCUSSION THREAD

**Discussion by asimmonsh** [hackatomik/SISCOx256#14] [Planear Curso | Lab]
Puede que sea la única o no, pero me da curiosidad saber si al final del semestre desarrollaremos un **examen semestral** o tendremos que entregar algún **proyecto** en vez del examen. En caso tal sea el proyecto (lo cuál creo es lo mas probable), ¿de qué trataría? Para prepararme mentalmente.

**Comment by tapilew** [comment]
Efectivamente, el profe siempre trabaja con proyectos. Lo más probable es que deban sustentarlo en el lobby FISC a final de semestre.

Te adelanto que uno de sus temas favoritos es ["economía plateada"](https://en.wikipedia.org/wiki/Silver_economy) (diseñar productos con las personas mayores en mente), pero por el momento que sepa el tema a trabajar este semestre es totalmente libre.

¿De qué trata entonces el proyecto? Principalmente aprender a trabajar en equipo con las herramientas que ofrece GitHub. Idealmente crear un repositorio donde lleven registro de su trabajo y que cada uno haya contribuido en él mediante "commits".

Dependiendo de su carrera o lo que haga sentido para ustedes pueden decidir hacer un proyecto de software (o incluso documentación en texto plano, markdown) con un tema que les llame la atención aprender. Ya sea relacionado a KnockKnock Server o completamente distinto. Tienen total libertad con tal que sea de ayuda para su futuro profesional.

Hace unos días compartí [esta discusión](https://github.com/hackatomik/SISCOx256/discussions/7) sobre una propuesta para trabajar así durante el semestre. Sería bueno que lo leas y comentes qué piensas o de qué tema necesitas aclaración. Es solo una propuesta así que si tienes una mejor idea para practicar el trabajo en equipo estaremos encantados de leer todos los comentarios y hacer los cambios necesarios de acuerdo a sus necesidades.
